bundle/bin/rke2-killall.sh

```diff
@@ -84,4 +84,9 @@ if [ -d /sys/class/net/nodelocaldns ]; then
 fi
 
 rm -rf /var/lib/cni/
+
+# Delete iptables created by CNI plugins or Kubernetes (kube-proxy)
 iptables-save | grep -v KUBE- | grep -v CNI- | grep -v cali- | grep -v cali: | grep -v CILIUM_ | grep -v flannel | iptables-restore
+echo
+echo if this cluster has been upgraded and was using canal, you may need to remove the flannel iptables with 'iptables -D POSTROUTING -s $cluster-cidr -j MASQUERADE --random-fully' and 'iptables -D POSTROUTING ! -s $cluster-cidr -d  -j MASQUERADE --random-fully'
```

Comment:
Might it be simpler to tell users to just reboot or manually clear all the iptables rules? If we want to print commands, lets at least give them a couple lines to copy-paste:
```suggestion
echo 'If this cluster was upgraded from an older release of the Canal CNI, you may need to manually remove some flannel iptables rules:'
echo -e '\texport cluster_cidr=YOUR-CLUSTER-CIDR'
echo -e '\tiptables -D POSTROUTING -s $cluster_cidr -j MASQUERADE --random-fully'
echo -e '\tiptables -D POSTROUTING ! -s $cluster_cidr -d  -j MASQUERADE --random-fully'
```

Reply:
Thanks for the suggestion. Yes, that would be simpler but rebooting might not be an option and if there are other tools using iptables, it might be difficult to find the exact rules. Copy-paste lines is a good idea